static/js/show-task.js

```diff
@@ -37,3 +37,28 @@ function saveTask() {
             );
     }});
 }
+
+function refresh() {
+    search_term = document.getElementById("search_by_task_name_form").search_term.value;
```

Comment:
For what it's worth, I hardly think performance would be an issue here -- it's not like we're calling this 1000s of times in a loop or anything, so it's fine to go with `$('#search_by_task_name_form')` for consistency if you want. No big deal if you prefer to leave as it is.

Reply:
I've changed it to the jquery format.